get-started.html

```diff
@@ -0,0 +1,66 @@
+---
+layout: default
+---
+<main class="site pad">
+  <div class="get-started">
+    <div class="get-started-heading">
+      <h2>Get Started</h2>
+    </div>
+
+    <div class="start">
+      <div class="start-block">
+        <div class="start-content">
+          <div class="left-column">
+            <h2 id="inform">
+              Inform yourself</h2>
+            <p>Grin is different than what you know and use every day. Before you start using Grin, there are a few things that you need to know in order to use it securely and avoid common pitfalls. Start with the basics. What is Grin? How is it used? How is it different than Bitcoin?
+              </p>
+            <div>
+              <a class="btn-bright" href="https://github.com/mimblewimble/grin/blob/master/doc/grin4bitcoiners.md">Read more</a>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="start-block">
+        <div class="start-content">
+          <div class="right-column">
+            <h2 id="choose">
+              Run your Node / Choose your wallet</h2>
+            <p>Grin makes it easy and fast to setup a full node. There is also a wide range of grin wallets available for all major operating systems and devices to serve a variety of your needs. Choosing a wallet is easy and can be done in minutes.</p>
+            <div>
+              <a class="btn-bright" href="/download">Download</a>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="start-block">
+        <div class="start-content">
+          <div class="left-column">
+            <h2 id="get">
+              Get Grin</h2>
+            <p>You can get Grin by accepting it as a payment for goods and services. You can also buy Grin on different exchanges.</p>
+            <div>
+              <a class="btn-bright" href="https://coinmarketcap.com/currencies/grin/#markets">Buy Grin</a>
```

Comment:
How about linking to https://github.com/mimblewimble/docs/wiki/Community-projects#exchanges instead?

Reply:
Indeed!